recipe_scrapers/instructables.py

```diff
@@ -0,0 +1,49 @@
+from ._abstract import AbstractScraper
+from ._grouping_utils import group_ingredients
+from ._utils import normalize_string
+
+
+class Instructables(AbstractScraper):
+    @classmethod
+    def host(cls):
+        return "instructables.com"
+
+    def ingredient_groups(self):
+        return group_ingredients(
+            self.ingredients(),
+            self.soup,
+            ".step-body p strong",
+            ".step-body li",
+        )
+
+    def title(self):
+        title_tag = self.soup.find("h2", class_="step-title sr-only")
+        return normalize_string(title_tag.get_text())
+
+    def author(self):
+        author_tag = self.soup.find("span", class_="author-name")
+        return normalize_string(author_tag.get_text().replace("By ", ""))
+
+    def ingredients(self):
+        ingredients_section = self.soup.find("section", id="stepsupplies")
+        if not ingredients_section:
+            return []
+
+        ingredients_tags = ingredients_section.find_all("li", class_="ql-indent-1")
+        return [normalize_string(tag.get_text()) for tag in ingredients_tags]
+
+    def instructions(self):
+        steps = self.soup.find_all("section", class_="step")
```

Comment:
Let's make sure to exclude/filter-out the `ingredients` (supplies) step, otherwise we're duplicating it in the instructions.